### PR TITLE
7747 OS acct instance events

### PR DESCRIPTION
**Pull request by rcordovano**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/CaseEventListener.java [hunk 1]
```diff
@@ -130,6 +133,14 @@ public void propertyChange(PropertyChangeEvent evt) {
                 jobProcessingExecutor.submit(new DataSourceNameChangedTask(dbManager, evt));
             }
             break;
+            case OS_ACCT_INSTANCES_ADDED: {
+                // STUB, TO BE REPLACED
```
Comment (markmckinnon) [inline Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/CaseEventListener.java:137]: SHould this comment be removed

Comment (rcordovano) [reply Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/CaseEventListener.java:137]: @markmckinnon, my intention was for you to remove it and replace my logging with your code. Does that work for you?

Comment (markmckinnon) [reply Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/CaseEventListener.java:137]: Yes it does.

Comment (rcordovano) [reply Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/CaseEventListener.java:137]: Cool.

**Review by markmckinnon — APPROVED** [review]

**Pull request merged**